.vscode/launch.json

```diff
@@ -0,0 +1,13 @@
+{
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
I didn't see any reason to not include it, in case someone wants to debug this later and doesn't want to figure it out theirself.